# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lablgtk/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur lablgtk-2.12.0.orig/src/ml_panel.c lablgtk-2.12.0/src/ml_panel.c
--- lablgtk-2.12.0.orig/src/ml_panel.c	2007-06-20 15:40:34.000000000 +0800
+++ lablgtk-2.12.0/src/ml_panel.c	2009-06-18 20:00:02.212296906 +0800
@@ -23,6 +23,7 @@
 #include <string.h>
 
 #include <libgnomeui/gnome-client.h>
+#include <libgnomeui/gnome-ui-init.h>
 #include <panel-applet.h>
 
 #include <caml/mlvalues.h>

